nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -37,6 +37,11 @@
           PostgreSQL now defaults to major version 13.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          spark now defaults to spark 3, updated from 2.
```

Comment:
Is there an upgrade guide we should link?

Reply:
There is a migration guide over at https://spark.apache.org/docs/latest/core-migration-guide.html#upgrading-from-core-24-to-30
I'll add this to the release notes.

Reply:
Perfect. Thank you!